src/main/java/com/easypost/model/Error.java

```diff
@@ -1,45 +1,46 @@
 package com.easypost.model;
 
+import java.util.List;
+
 public final class Error {
-    private String field;
     private String message;
-    private String suggestion;
     private String code;
+    private List<Error> errors;
 
     /**
-     * Get the error code.
+     * Get the errors from an Error object.
      *
-     * @return code
+     * @return errors
      */
-    public String getCode() {
-        return code;
+    public List<Error> getError() {
+        return errors;
     }
 
     /**
-     * Set the error code.
-     *
-     * @param code code
+     * Set the errors for an Error object.
+     * 
+     * @param errors errors
      */
-    public void setCode(final String code) {
-        this.code = code;
+    public void setError(List<Error> errors) {
```

Comment:
Issue: This must be plural since it's a list of multiple errors

```suggestion
    public void setErrors(List<Error> errors) {
```